### PR TITLE
KEYCLOAK-9710 RHSSO CD 6 adjustments

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/KEYCLOAK-9710

### CODE REVIEW

File: override-dev-local.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ osbs:
             name: containers/redhat-sso-7
             branch: jb-sso-cd-dev-rhel-7
 artifacts:
-      - sha256: d8f7733cad3f8cc0daa14bfd70b1c6e1b116744baef67fd71bb02249e590f8f2
+      - md5: c6c5690dd226280b612eed960d66a2f9
```
Comment (martin-kanis) [inline override-dev-local.yaml:6]: I think we don't have change override-dev-local.yaml. This is used for local builds only I think.

File: image.yaml [hunk 2]
```diff
@@ -1,25 +1,25 @@
 schema_version: 1
 
 name: "redhat-sso-7-tech-preview/sso-cd"
-description: "Red Hat Single Sign-On 7.4 continuous delivery container image"
-version: "7.4.0.CD05"
+description: "Red Hat Single Sign-On continuous delivery container image"
+version: "7.4.0.CD06"
```
Comment (hmlnarik) [inline image.yaml]: "6"

File: override-dev-local.yaml (hunk 1 above)
Comment (slaskawi) [reply override-dev-local.yaml:6]: The old artefact was for EAP CD15, so we need to change it.

File: image.yaml (hunk 2 above)
Comment (slaskawi) [reply image.yaml]: Fixed, thanks!

**Review by hmlnarik — APPROVED** [review]

File: README.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Red Hat Single Sign-On 7.2 container image
+# Red Hat Single Sign-On container image
```
Comment (iankko) [inline README.adoc:1]: @slaskawi To be consistent, I would propose to use what's in the container catalog:
- https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/redhat-sso-7-tech-preview/sso-cd-openshift

Thus `Red Hat Single Sign-On Continuous Delivery (RH-SSO CD) for OpenShift` IMHO.

File: image.yaml [hunk 4]
```diff
@@ -1,25 +1,25 @@
 schema_version: 1
 
 name: "redhat-sso-7-tech-preview/sso-cd"
-description: "Red Hat Single Sign-On 7.4 continuous delivery container image"
-version: "7.4.0.CD05"
+description: "Red Hat Single Sign-On continuous delivery container image"
```
Comment (iankko) [inline image.yaml]: Same here? IOW replace with `Red Hat Single Sign-On Continuous Delivery (RH-SSO CD) for OpenShift`

**Review by iankko — COMMENTED** [review]
Besides the above two comments, the rest LGTM.

File: README.adoc (hunk 3 above)
Comment (slaskawi) [reply README.adoc:1]: I'm not sure if this is right to be honest. This is a standalone image and it doesn't contain OpenShift related bits (such as `KUBE_PING`). So saying it's for OpenShift is not quite right. 

Let's leave it with minimum set of changes...

File: image.yaml (hunk 4 above)
Comment (slaskawi) [reply image.yaml]: I changed this to `Continuous Delivery`

File: README.adoc (hunk 3 above)
Comment (iankko) [reply README.adoc:1]: @slaskawi Right, you are correct with the `OpenShift` piece. Ok, let's keep the current version.

**Review by iankko — APPROVED** [review]
LGTM

**Pull request merged**